srcpkgs/python3-anyio/template

```diff
@@ -11,8 +11,8 @@ maintainer="dkwo <nicolopiazzalunga@gmail.com>"
 license="MIT"
 homepage="https://github.com/agronholm/anyio"
 distfiles="${PYPI_SITE}/a/anyio/anyio-${version}.tar.gz"
-checksum=41c4be842c284222b197a625d76a7ab85adf9d52788f563172fe180c2744b6c1
-# Tests require unpackaged "trustme"
```

Comment:
Is there a reason we can't package `trustme`?

Reply:
No particular reason.
Actually, I packaged it yesterday, and then `anyio` gave some other warning that tests are not supported and will be dropped in a future release, so I left it as it is.